python/examples/rdm_snapshot.py

```diff
@@ -419,9 +419,9 @@ def _RDMRequestComplete(self, response, unpacked_data, unpack_exception):
 
 
 def Usage():
+  print("Usage: rdm_snapshot.py --universe <universe> [--input <file>] "
+        "[--output <file>]\n")
   print(textwrap.dedent("""\
-  Usage: rdm_snapshot.py --universe <universe> [--input <file>] [--output <file>]
```

Comment:
I'll stick with the current mix then, I think this is cleaner than backslash, but probably not in the middle of a block where I'd need to split it into three print statements.